group/filippo_ed25519/suite.go

```diff
@@ -0,0 +1,71 @@
+package filippo_ed25519
+
+import (
+	"crypto/cipher"
+	"crypto/sha256"
+	"hash"
+	"io"
+	"reflect"
+
+	"go.dedis.ch/fixbuf"
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/group/internal/marshalling"
+	"go.dedis.ch/kyber/v3/util/random"
+	"go.dedis.ch/kyber/v3/xof/blake2xb"
+)
+
+// SuiteFilippoEd25519 implements some basic functionalities such as Group, HashFactory,
+// and XOFFactory.
+type SuiteFilippoEd25519 struct {
+	Curve
+	r cipher.Stream
+}
+
+// Hash returns a newly instanciated sha256 hash function.
+func (s *SuiteFilippoEd25519) Hash() hash.Hash {
+	return sha256.New()
+}
+
+// XOF returns an XOF which is implemented via the Blake2b hash.
+func (s *SuiteFilippoEd25519) XOF(key []byte) kyber.XOF {
+	return blake2xb.New(key)
```

Comment:
```suggestion
func (s *SuiteFilippoEd25519) XOF(seed []byte) kyber.XOF {
	return blake2xb.New(seed)
```

for coherence with the rest of Kyber